src/LibraryViewExtensionMSWebBrowser/web/library/library.html

```diff
@@ -229,8 +260,126 @@
             libraryItemsContainer.style.zoom = dpiScale;
         }
 
+        //This will find a specific div in the html and then it will apply the glow animation on it
+        function highlightLibraryItem(itemName, enableHighlight) {
+            var found_div = null;
+            var libraryItemsText = document.getElementsByClassName("LibraryItemText");
+            for (var i = 0; i < libraryItemsText.length; i++) {
+                if (libraryItemsText[i].textContent == itemName) {
+                    found_div = libraryItemsText[i].parentNode;
+                    break;
+                }
+            }
+            if (found_div != null) {
+                if (enableHighlight == true) {
+                    glowAnimation(found_div, true);
+                }
+                else {
+                    glowAnimation(found_div, false);
+                }
+            }
+        }
+
+        //This will execute(or stop) the glow animation in a specific <div> based in the enable parameter
+        function glowAnimation(divElement, enable) {
+            if (enable == true) {
```

Comment:
I guess it would be simpler if we just `if(enable)`. What do you think?

Reply:
@filipeotero  This change and another one that where using the same syntaxis were updated in the next commit:
https://github.com/DynamoDS/Dynamo/pull/12312/commits/2933fbf31bc87989ba49d35636b97c904c986d5f